src/main/java/eu/hansolo/tilesfx/skins/DonutChartTileSkin.java

```diff
@@ -209,7 +209,7 @@ private void drawChart() {
 
         // Sum
         if (tile.isValueVisible()) {
-            chartCtx.setFont(Fonts.latoRegular(canvasSize * 0.15));
+            chartCtx.setFont(Fonts.latoRegular(canvasSize * 0.12));
```

Comment:
If you can keep the font size at canvasSize * 0.15 I will merge the request

Reply:
Yes sure, no preoblem.

Reply:
thx